doc/plugin_server_notifier_k8sbundle.md

```diff
@@ -134,3 +135,49 @@ server to
         }
     }
 ```
+
+### Multipe clusters
+
+
+```
+    Notifier "k8sbundle" {
+        plugin_data {
+          namespace = "infra"
+          config_map = "agents"
+          config_map_key = "bootstrap.crt"
+          kube_config_file_path = "/path/to/cluster1-kubeconfig"
+
+          clusters  = [
+            {
+                namespace = "infra"
+                config_map = "agents"
+                config_map_key = "bootstrap.crt"
+                kube_config_file_path = "/path/to/clustrer2-kubeconfig"
```

Comment:
`kube_config_file_path` should be required when defined here, which is a departure from the documented behavior. It probably makes sense to leave a config comment here noting this.

Reply:
That's the one field I wasn't sure about. But since I'm biased and trying to use this on the same cluster but with multiple namespaces, I thought It might make sense not to require that field. 

That particular use case looks like this 

```
plugin_data {
      namespace = "spire"          
      clusters  = [
        {
          namespace = "istio-system"              
        },
        {
          namespace = "default"              
        }
      ]
}
```